src/YT/YT_SetParameter.cpp

```diff
@@ -108,10 +108,16 @@ void YT_SetParameter( const int NPatchAllLv, const int NField, const int NPatchL
    if ( yt_set_UserParameterDouble("gamma",    1, &gamma   ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_set_UserParameterDouble() set %s failed !!\n", "GAMMA" );
    if ( yt_set_UserParameterDouble("mu",       1, &mu      ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_set_UserParameterDouble() set %s failed !!\n", "MOLECULAR_WEIGHT" );
    if ( yt_set_UserParameterInt   ("srhd",     1, &srhd    ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_set_UserParameterInt() set %s failed !!\n", "SRHD" );
-#  endif
+
+#  elif ( MODEL == ELBDM )
+   const int srhd = 0;
+   if ( yt_set_UserParameterInt   ("srhd",     1, &srhd    ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_set_UserParameterInt() set %s failed !!\n", "SRHD" );
+#  endif // MODEL
+
```

Comment:
[Bug] According to the latest yt version (4.4.0), `mu` is always defined ([here](https://github.com/yt-project/yt/blob/main/yt/frontends/gamer/data_structures.py#L360)) and will be used to set fluid field ([here](https://github.com/yt-project/yt/blob/main/yt/frontends/gamer/fields.py#L70)). The current libyt code in ELBDM does not contain `mu`.
This bug is discovered by @hsinhaoHHuang and @koarakawaii.